### PR TITLE
Edit form (basic)

### PR DESCRIPTION
**Pull request by kisiro55**

### CODE REVIEW

File: app/controllers/practices_controller.rb [hunk 1]
```diff
@@ -8,7 +8,7 @@ def index
       @practices = Practice.where(:practice_type => params[:practice_type])
       # flash[:notice] = "There are <b>#{@practices.count}</b> practices in this category".html_safe
     else
-       @practices = Practice.all
+       @practices = Practice.where(user: current_user)
```
Comment (Lastimoso) [inline app/controllers/practices_controller.rb:11]: See other PR comments:
This is an important convension to follow: `current_user.practices`

File: app/controllers/practices_controller.rb [hunk 2]
```diff
@@ -47,14 +48,13 @@ def destroy
...
-
 private
 
   def practice_params
     params.require(:practice).permit(:invoice_first_name, :invoice_last_name, :invoice_email, :additional_email, :phone, :invoice_phone,:session_rate, :hourly_rate, :pension_work_default, :type_pension_form, :agency_name)
   end
 
-    def set_practice
+  def set_practice
     @practice = Practice.find(params[:id])
```
Comment (Lastimoso) [inline app/controllers/practices_controller.rb:58]: Again, same leaky and dangerous practices. Use the association to scope from current_user

File: app/controllers/practices_controller.rb (hunk 1 above)
Comment (kisiro55) [reply app/controllers/practices_controller.rb]: Thanks Gerardo. I am doing this change in the next commit called "answering comments on PR to current_user_practices" (9ada8df)

Note: it was first giving an error, so i have to add the following line on the practice model: 

`has_many :practices`

File: app/controllers/practices_controller.rb (hunk 2 above)
Comment (kisiro55) [reply app/controllers/practices_controller.rb]: already addressed on the same commit than the previous one.

**Pull request merged**